primitives/io/src/lib.rs

```diff
@@ -460,7 +498,8 @@ pub trait Crypto {
 		msg: &[u8],
 		pub_key: &ed25519::Public,
 	) -> bool {
-		ed25519::Pair::verify(sig, msg, pub_key)
+		// ed25519::Pair::verify(sig, msg, pub_key)
```

Comment:
Please change the state of this PR to Draft since it seems it's being used to experiment with stuff and for sure is not ready to be merged to Substrate.